packages/rum-core/test/dom.ts

```diff
@@ -28,3 +30,25 @@ export function createIsolatedDom() {
     },
   }
 }
+
+export function appendElement(tagName: string, attributes: { [key: string]: string }) {
```

Comment:
<!-- Suggestions are specific requests to improve the subject under review. It is assumed that we all want to do what's best, so these comments are never dismissed as “mere suggestions”, but are taken seriously.  -->
**💬 suggestion:** ‏I think this kind of utility is great, but it duplicates a bit the use-cases of `createIsolatedDom`. It could be great if we kept one tool for the job, but I don't think the current `appendElement` implementation is practical enough to replace the `createIsolatedDom` usages in the future.

What about something like this:

```ts
function useIsolatedDom(html: string) {
  const dom = createIsolatedDom()
  registerCleanupTask(() => dom.clear())
  return dom.append(html)
}
//
const div = useIsolatedDom('<div id="toto">')
const text = useIsolatedDom('toto') // createIsolatedDom could be modified a bit to support this use-case
```

then we could progressively remove `createIsolatedDom` direct usages.

Reply:
It's true. I would also prefer a single API. My only concern is that isolatedDom
- uses `innerHTML`. Ideally I'd like to use this new API also in Replay and that may be a problem. 
- it uses an iframe which isolate the DOM and the event triggered inside it which may cause problem for some tests 
Wdyt?

Reply:
Like discussed. I'll make a following PR to harmonize the DOM APIs.